tests/virt/cluster/vm_cloning/test_vm_cloning.py

```diff
@@ -304,7 +304,7 @@ def test_check_new_smbios_serial_on_clone(self, fedora_target_vm_instance):
         )
 
     @pytest.mark.polarion("CNV-10320")
-    def test_migrate_the_vm_clone(self, skip_when_one_node, skip_if_no_common_cpu, fedora_target_vm):
+    def test_migrate_the_vm_clone(self, skip_if_no_common_cpu, fedora_target_vm):
```

Comment:
here and in many other places - if you remove `skip_when_one_node` it means we are not going to run the full group of these tests on SNO cluster? 
All VM cloning tests should run on sno cluster (except migration of course)

Reply:
I double checked. Currently none of these tests have no sno marker. So none of them are running on sno cluster as of now. The teams should mark the tests that requires SNO. If a file has sno marker, I am updating tests individually when I remove `skip_when_one_node`. But if the marker is not present, I will leave it for the team to mark the tests on their own.